actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -658,18 +659,24 @@ public void setDeactivationTimeout(long deactivationTimeoutMs)
 
         localObjectsCleaner.setActorDeactivationExtensions(getAllExtensions(ActorDeactivationExtension.class));
 
-        final ResponseCaching cacheManager = new ResponseCaching();
+        final ResponseCachingExtension cacheManager = getAllExtensions(ResponseCachingExtension.class)
+                .stream()
+                .findFirst()
```

Comment:
If more than one extension is provided, the behaviour here might be confusing.   Maybe it should just fail-fast if there's more than one found?